src/calculator.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+
+var Precedence = require('./precedence');
```

Comment:
The convention in all programming languages is to import all other modules at the top of the file. So here you would have:
```
var Screen = require('./classScreen');
var Precedence = require('./precedence');
var Memory = require('./classMemory');
```
And then you would use them in rest of the file.